domains/TdoubleT.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "tuanthu1",
+        "email": "hoangtuanthu981@gmail.com"
+    },
+    "record": {
+        "RECORDTYPE": "https://tuanthu1.github.io/tuanthu2"
```

Comment:
RECORDTYPE should be filled with an actual record type. See docs: https://docs.is-a.dev